controllers/topology/builtin.go

```diff
@@ -230,6 +267,79 @@ func GetRoles(obj ObjectWithMeta) ([]string, error) {
 	return nil, errors.New("failed to parse roles from annotations")
 }
 
+// Harness of edit topology query
+func (s *BuiltInTopologyService) EditTopology(replicas map[string]*corev1.Pod, rsetName string) error {
+	var replicasUri []EditTopologyServer
+	var replicasetsValues Replicasets
+	var replicasetsValuesArray []Replicasets
+	var lastNode *corev1.Pod
+
+	// Generating URI's array
+	for key, pod := range replicas {
+		fmt.Println("Recieved values to join: ", key)
+		thisPodLabels := pod.GetLabels()
+		clusterDomainName, ok := thisPodLabels["tarantool.io/cluster-domain-name"]
+		if !ok {
+			clusterDomainName = "cluster.local"
+		}
+
+		advURI := fmt.Sprintf("%s.%s.%s.svc.%s:3301",
+			pod.GetObjectMeta().GetName(),      // Instance name
+			s.clusterID,                        // Cartridge cluster name
+			pod.GetObjectMeta().GetNamespace(), // Namespace
+			clusterDomainName)                  // Cluster domain name
```

Comment:
Please put a closing parenthesis in the next line

```suggestion
		advURI := fmt.Sprintf("%s.%s.%s.svc.%s:3301",
			pod.GetObjectMeta().GetName(),      // Instance name
			s.clusterID,                        // Cartridge cluster name
			pod.GetObjectMeta().GetNamespace(), // Namespace
			clusterDomainName,                  // Cluster domain name
		)
```